lib/widgets/content.dart

```diff
@@ -752,22 +778,27 @@ final _kCodeBlockStyle = kMonospaceTextStyle
 // const _kInlineCodeRightBracket = '⟩';
 
 class UserMention extends StatelessWidget {
-  const UserMention({super.key, required this.node});
+  const UserMention({
+    super.key,
+    required this.surroundingTextStyle,
+    required this.node,
+  });
 
+  final TextStyle surroundingTextStyle;
   final UserMentionNode node;
 
   @override
   Widget build(BuildContext context) {
     return Container(
       decoration: _kDecoration,
-      padding: const EdgeInsets.symmetric(horizontal: 0.2 * kBaseFontSize),
+      padding: EdgeInsets.symmetric(horizontal: 0.2 * surroundingTextStyle.fontSize!),
```

Comment:
Interesting, it looks like the revision I previously reviewed didn't have this change. Nor the similar padding change for dates, or the one for the clock icon size.

Would you update the screenshot to reflect this revision?

Reply:
Ah, yeah. Done.

Reply:
Cool, thanks.

Looking at that screenshot, I think it's probably best to go ahead and make this adjustment for the clock icon, but *not* for the padding. The expanded padding feels kind of out of scale (even though in a literal sense it is precisely to scale).

It's reminiscent of the situation described at https://github.com/zulip/zulip-mobile/blob/dff4d4d5ed495aba0e97f105215ce425909e6d53/docs/background/webview.md#why-mostly-px-and-not-rem :
> Typical advice for web design is to use `rem` or `em` for nearly all lengths. This means that when the font size grows or shrinks, not only the text but the whole layout scales with it, including padding, margin, and images.
>
> But on mobile, if you go to a device's settings and [change the font size](https://github.com/zulip/zulip-mobile/blob/dff4d4d5ed495aba0e97f105215ce425909e6d53/docs/background/layout.md), you'll find this is not at all what apps do. When you make the font size giant, the text in an app grows with it -- but everything that _isn't_ text stays the same. Icons in the UI; user avatars; padding around text; padding or margin between elements; indentation of items in a list; all stay exactly the same. This is true on Android and iOS, and of flagship apps from Google, Apple, Facebook, and others.

Arguably that'd call for not scaling the clock icon either; but I think the clock icon looks best when it does get scaled. Effectively it feels like part of the text — like a sort of custom emoji.